projects/sienna/index.js

```diff
@@ -16,23 +20,25 @@ const SIENNA_SINGLE_SIDED_POOLS = [
 const SIENNA_TOKEN_ADDRESS = "secret1rgm2m5t530tdzyd99775n6vzumxa5luxcllml4";
 const LEND_OVERSEER_CONTRACT = null;
 
-const SECRET_NODE_URL = "https://bridgeapi.azure-api.net/";
+const SECRET_NODE_URL = "https://bridgeapi.azure-api.net/proxy/";
 const queryClient = new CosmWasmClient(SECRET_NODE_URL);
 
 const CACHED_TOKENS = {};
 
 async function Pairs() {
-    return (await PairsV1()).concat(await PairsV2());
-}
-
-async function PairsV1() {
-    const pairs = await queryClient.getContracts(pairCodeId);
-    return pairs.filter((p) => p.label.endsWith(`${factoryContract}-${pairCodeId}`));
-}
-
-async function PairsV2() {
-    const pairs = await queryClient.getContracts(pairCodeIdV2);
-    return pairs.filter((p) => p.label.endsWith(`${factoryContractV2}-${pairCodeIdV2}`));
+    return new Promise((resolve) => {
+        mapLimit(factories, 1, async (factory) => {
+            try {
+                const result = await queryClient.getContracts(factory.code);
+                const pairs = result.filter((p) => p.label.endsWith(`${factory.address}-${factory.code}`));
+                return Promise.resolve(pairs);
+            } catch (e) {
```

Comment:
Hi, can you revert the error handling, we prefer adapter failing over returning wrong tvl

Reply:
Understood, reverted the error handling

Reply:
thanks for fixing it quickly